examples/esp-idf-component/sdkconfig.defaults

```diff
@@ -4,6 +4,7 @@ CONFIG_ESP32_DEFAULT_CPU_FREQ_240=y
 CONFIG_ESP_SYSTEM_EVENT_TASK_STACK_SIZE=4096
 CONFIG_PTHREAD_TASK_STACK_SIZE_DEFAULT=8192
 CONFIG_ESP_TLS_SERVER=y
+CONFIG_LWIP_MAX_SOCKETS=13
```

Comment:
Now that we have a fix for https://viam.atlassian.net/browse/RSDK-9213 in via #355, I'm more comfortable bumping this up a little bit. Arguably, we can have more sockets now since you may have both local WebRTC and app brokered running simultaneously.